parsers/ES_CN.py

```diff
@@ -0,0 +1,81 @@
+# The arrow library is used to handle datetimes
+from arrow import get, utcnow
+# The request library is used to fetch content through HTTP
+from requests import Session
+from reescraper import CanaryIslands, NoDataException, TimestampException
+
+
+def fetch_consumption(country_code='ES-CN', session=None):
+    if not session:
+        session = Session
+
+    try:
+        response = CanaryIslands(session).get()
+    except NoDataException:
+        response = None
+    except TimestampException:
+        response = None
+
+    if not response:
+        datetime = utcnow().datetime
+        consumption = None
+    else:
+        datetime = get(response.timestamp).datetime
+        consumption = response.demand
+
+    data = {
+        'countryCode': country_code,
+        'datetime': datetime,
+        'consumption': consumption,
+        'source': 'https://demanda.ree.es/movil'
+    }
+
+    return data
+
+
+def fetch_production(country_code='ES-CN', session=None):
+
+    if not session:
+        session = Session
+
+    try:
+        response = CanaryIslands(session).get()
+    except NoDataException:
+        response = None
+    except TimestampException:
+        response = None
+
+    if not response:
+        datetime = utcnow().datetime
+    else:
+        datetime = get(response.timestamp).datetime
+
+    data = {
+        'countryCode': country_code,
+        'datetime': datetime,
+        'production': {},
```

Comment:
If we know that coal, biomass, nuclear or geothermal is 0, can you then set them here?